solr/core/src/java/org/apache/solr/cli/AuthTool.java

```diff
@@ -96,7 +98,7 @@ public class AuthTool extends ToolBase {
           .argName("FILE")
           .required()
           .desc(
-              "This is where any authentication related configuration files, if any, would be placed.")
+              "This is where any authentication related configuration files, if any, would be placed.  Defaults to $SOLR_HOME.")
```

Comment:
If I am not mistaken, this defaults to `SOLR_HOME` only when `AuthTool` is used via the CLI script, but if directly used it does not default to any path. Is that correct?

I think we had a discussion before about using the classes without the CLI scripts, but I don't remember anymore if that was a valid scenario.

Reply:
I don't think using these outside the CLI scripts is actually a true scenario...   In fact, I think if folks are using these tools outside of the bin/solr scripts (like some folks have their own start scripts) it points to weaknesses in bin/solr that we need to improve!